<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telegraph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/ico</normaloff>:/image/ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>新建群聊</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>充值</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/image/cash.png</normaloff>:/image/cash.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>修改资料</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/image/C:/Users/i/Downloads/profile 2.png</normaloff>:/image/C:/Users/i/Downloads/profile 2.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>个性化</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/image/C:/Users/i/Downloads/paint_palette.png</normaloff>:/image/C:/Users/i/Downloads/paint_palette.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>图片</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/image/C:/Users/i/Downloads/文档类型-图片.png</normaloff>:/image/C:/Users/i/Downloads/文档类型-图片.png</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_10">
            <property name="text">
             <string>文件</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/image/C:/Users/i/Downloads/文件.png</normaloff>:/image/C:/Users/i/Downloads/文件.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>表情</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/image/C:/Users/i/Downloads/bqb.png</normaloff>:/image/C:/Users/i/Downloads/bqb.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_11">
            <property name="text">
             <string>红包</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/avatar/红包.png</normaloff>:/avatar/红包.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>抖动</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/image/C:/Users/i/Downloads/铃铛带圈.png</normaloff>:/image/C:/Users/i/Downloads/铃铛带圈.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>清屏</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/image/C:/Users/i/Downloads/ic_cleaning_junk.png</normaloff>:/image/C:/Users/i/Downloads/ic_cleaning_junk.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_12">
            <property name="text">
             <string>语音</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/image/麦克风ico.png</normaloff>:/image/麦克风ico.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPlainTextEdit" name="textEdit_2"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>发送</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/image/C:/Users/i/Downloads/纸飞机.png</normaloff>:/image/C:/Users/i/Downloads/纸飞机.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="refresh">
   <property name="text">
    <string>刷新列表</string>
   </property>
   <property name="toolTip">
    <string>刷新在线用户列表</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
